src/disrupt.sh

```
@@ -23,15 +23,18 @@ if [[ ! -d "${IMG_ASSET_DIR}" || ! $(find "${IMG_ASSET_DIR}" -mindepth 1 -iname
     DICE_ROLL=0
 fi
 
-# add the image if necessary
-if [ "${DICE_ROLL}" -eq 1 ]; then
-    RANDOM_IMAGE=$(find "${IMG_ASSET_DIR}" -type f | shuf -n 1)
-
-    cat "${MSG_TEMPLATE_IMG}" >> "${TEMP_FILE}"
-    base64 "${RANDOM_IMAGE}" >> "${TEMP_FILE}"
+#Check for an image associated with the fact
+ASSOCIATED_IMAGE=$("${RUN_DIR}/src/lf_imap.py ${RANDOM_FACT}}")
+if [ "$ASSOCIATED_IMAGE" ] && [ -s "$ASSOCIATED_IMAGE" ]; then
```

So is the plan basically to have specific facts always send an image?

Would it make sense to keep this under the dice roll, and then use the mapped image if one exists, falling back to the random image when there isn't one?

---

Actually that might make more sense... then it might be rare to actually see the associated image though and I think the motivation was to have certain facts always get an image. @Hazematman ?